python/TestHarness/testers/RunCommand.py

```diff
@@ -0,0 +1,30 @@
+import re, os, sys, time
+from Tester import Tester
+from RunParallel import RunParallel # For TIMEOUT value
+
+class RunCommand(Tester):
+
+  @staticmethod
+  def validParams():
+    params = Tester.validParams()
+    params.addRequiredParam('command',      "The command line to execute for this test.")
```

Comment:
Yeah yeah. Same thing is/was possible with `post_command` and committing crap to `Makefile`s. It's an old hat unless you have a new idea how to prevent it.